tests/std/tests/GH_002030_asan_annotate_string/env.lst

```diff
@@ -2,9 +2,9 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 # This test matrix is the usual test matrix, with all currently unsupported options removed, crossed with the ASan flags.
-# TRANSITION, VSO-1350252
-# Due to a bug in the ASan libs using ASan with /MD or /MT requires IDL==0 and using /MDd or /MTd requires IDL==2.
-# clang-cl does not currently support targeting /MDd or /MTd.
+
+# TRANSITION, VSO-1350252 - due to vcasan.lib including the standard library, we can't use it (pending 17.5 preview 2).

```

Comment:
Does "it" mean "the standard library" or "vcasan.lib"? Is this waiting for VSO-1350252 to be fixed, or for 17.5 preview 2 to release?

Reply:
vcasan.lib; this is waiting for whatever release fixed VSO-1350252.